react/store-global.css

```diff
@@ -31,43 +63,41 @@ body {
   }
 }
 
-/*
-  Media Query Extensions:
-  -s  = 20em
-  -ns  = 40em
-  -m  = 40em
-  -l  = 64em
-  -xl  = 80em
-*/
+/* Make divs behave evenly */
+.vtex-store__template > div {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
 
-.vtex-page-padding {
-  padding-left: 0;
-  padding-right: 0;
+.vtex-store__template > div > div:last-child {
```

Comment:
Add a comment here to explain what this is trying to target.

Also, if it it could be explicitly target by a class, it would be better. The dependency would be clearer.

Reply:
Those styles was on  another component (footer). After checking, this definitions was unecessary.